server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDumpTransactionOptionsCheckUtility.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package org.eclipse.lsp.cobol.implicitDialects.cics.utility;
+
+        import org.antlr.v4.runtime.ParserRuleContext;
+        import org.eclipse.lsp.cobol.common.dialects.DialectProcessingContext;
+        import org.eclipse.lsp.cobol.common.error.ErrorSeverity;
+        import org.eclipse.lsp.cobol.common.error.SyntaxError;
+        import org.eclipse.lsp.cobol.implicitDialects.cics.CICSLexer;
+        import org.eclipse.lsp.cobol.implicitDialects.cics.CICSParser;
+
+        import java.util.HashMap;
+        import java.util.List;
+        import java.util.Map;
+
+        import static org.eclipse.lsp.cobol.implicitDialects.cics.CICSParser.*;
+
+/** Checks CICS Dump Transaction for required and invalid options */
+public class CICSDumpTransactionOptionsCheckUtility extends CICSOptionsCheckBaseUtility {
+
+    public static final int RULE_INDEX = RULE_cics_dump;
+
+    private static final Map<Integer, ErrorSeverity> DUPLICATE_CHECK_OPTIONS =
+            new HashMap<Integer, ErrorSeverity>() {
+                {
+                    put(CICSLexer.DUMP, ErrorSeverity.ERROR);
+                    put(CICSLexer.TRANSACTION, ErrorSeverity.ERROR);
+                    put(CICSLexer.FROM, ErrorSeverity.ERROR);
+                    put(CICSLexer.LENGTH, ErrorSeverity.ERROR);
+                    put(CICSLexer.FLENGTH, ErrorSeverity.ERROR);
+                    put(CICSLexer.COMPLETE, ErrorSeverity.ERROR);
+                    put(CICSLexer.TRT, ErrorSeverity.ERROR);
+                    put(CICSLexer.SEGMENTLIST, ErrorSeverity.ERROR);
+                    put(CICSLexer.LENGTHLIST, ErrorSeverity.ERROR);
+                    put(CICSLexer.NUMSEGMENTS, ErrorSeverity.ERROR);
+                    put(CICSLexer.TASK, ErrorSeverity.ERROR);
+                    put(CICSLexer.STORAGE, ErrorSeverity.ERROR);
+                    put(CICSLexer.PROGRAM, ErrorSeverity.ERROR);
+                    put(CICSLexer.TERMINAL, ErrorSeverity.ERROR);
+                    put(CICSLexer.TABLES, ErrorSeverity.ERROR);
+                    put(CICSLexer.FCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.PCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.PPT, ErrorSeverity.ERROR);
+                    put(CICSLexer.SIT, ErrorSeverity.ERROR);
+                    put(CICSLexer.TCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.DUMPID, ErrorSeverity.ERROR);
+                }
```

Comment:
Warning for all unparametrized options

Reply:
and conflicts

Reply:
should adressed at :[df409f7](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2577/commits/df409f736fab68af7185c967043caa596e5e40bc)